conversions/decimal_to_binary.py

```diff
@@ -35,9 +34,9 @@ def decimal_to_binary(num):
         True
     """
     assert type(num) in (int, float) and num == int(num)
-
+    num = int(num)
```

Comment:
This fails...
```
>>> decimal_to_binary(2.0) == bin(2.0)
True
```

Reply:
@cclauss bin(2.0) raises exception. And my code also raises exception. But the test still fails. Any idea how to resolve this issue? Can I just hard code the error in the doctests?
`**********************************************************************
File "decimal_to_binary.py", line 34, in __main__.decimal_to_binary
Failed example:
    decimal_to_binary(2.0) == bin(2.0) # doctest: +ELLIPSIS
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python3.6/doctest.py", line 1330, in __run
        compileflags, 1), test.globs)
      File "<doctest __main__.decimal_to_binary[9]>", line 1, in <module>
        decimal_to_binary(2.0) == bin(2.0) # doctest: +ELLIPSIS
      File "decimal_to_binary.py", line 38, in decimal_to_binary
        raise TypeError("'float' object cannot be interpreted as an integer")
    TypeError: 'float' object cannot be interpreted as an integer
**********************************************************************
`